lib/search/format-package-stream.js

```diff
@@ -0,0 +1,188 @@
+'use strict'
+
+var ms = require('mississippi')
+var jsonstream = require('JSONStream')
+var columnify = require('columnify')
+
+module.exports = function (opts) {
```

Comment:
Functions feel less lonely with names (also, the CLI's code base has a house style of using

```js
module.exports = hoistedFunctionName
function hoistedFunctionName () {
//...
```
)